src/acceptance-test/java/net/consensys/orion/acceptance/send/receive/DualNodesSendReceiveTest.java

```diff
@@ -76,22 +82,29 @@ void setUpDualNodes(@TempDirectory final Path tempDir) throws Exception {
     final Path key1key = copyResource("key1.key", tempDir.resolve("key1.key"));
     final Path key2pub = copyResource("key2.pub", tempDir.resolve("key2.pub"));
     final Path key2key = copyResource("key2.key", tempDir.resolve("key2.key"));
+    final Path key3pub = copyResource("key3.pub", tempDir.resolve("key3.pub"));
```

Comment:
Can this be simplified a bit? It's bit hard to read with some many keys here. Perhaps a function that copies keys that you provide a list of key names to? Something like copyKeys("key1", "key2", "key3", "key4", "key5")

Reply:
Have a look if you think that this is better ...